src/chainparams.h

```diff
@@ -85,7 +85,7 @@ class CChainParams
     const ChainTxData& TxData() const { return chainTxData; }
     int PoolMaxTransactions() const { return nPoolMaxTransactions; }
     int FulfilledRequestExpireTime() const { return nFulfilledRequestExpireTime; }
-    std::string SporkPubKey() const { return strSporkPubKey; }
+    const std::string &SporkAddress() const { return strSporkAddress; }
```

Comment:
nit: Should the ampersand be aligned to the left of the space here per updated style guidelines?